plugins/ansiblevault/ansible_vault.go

```diff
@@ -0,0 +1,25 @@
+package ansiblevault
```

Comment:
I'd call the plugin `ansible`. The `ansible-vault` CLI is shipped as part of the `ansible` package.